cniconf/watcher.go

```diff
@@ -128,3 +137,27 @@ func (w *watcher) apply(network metadata.Network, host metadata.Host) error {
 
 	return lastErr
 }
+
+func checkMTU(config interface{}) error {
+	props, _ := config.(map[string]interface{})
+	bridgeName, _ := props["bridge"].(string)
+	cniMTU := fmt.Sprintf("%.0f", props["mtu"])
+	c, err := events.NewDockerClient()
+	if err != nil {
+		return err
+	}
+
+	networks, err := c.ListNetworks()
+	if err != nil {
+		return err
+	}
+	for _, n := range networks {
```

Comment:
Can you please split the functionality of getting the MTU from docker as a separate function?
`func GetDockerNetworkBridgeMTU(string bridgeName)`